### PR TITLE
Step missing in procedure

### PR DESCRIPTION
**Pull request by joinimran**
There was a step missing that has been added in the document after confirmation from the portal.

Problem: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/749

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/manage-yammer-users/office-365-sign-in.md [hunk 1]
```diff
@@ -65,12 +65,14 @@ By default, when users select **Conversations** in SharePoint, they see their th
 1. In Office 365, go to **Admin** \> **SharePoint**.
     
 2. Click **Settings**.
+
+3. Click on **classic settings page**.
```
Comment (JohanFreelancer9) [inline yammer/manage-yammer-users/office-365-sign-in.md]:
```suggestion
3. Choose **Classic settings page**.
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@TeresaFG-writer - PR has been copy edited and is ready for final review. This is based on issue #749

Thank you.

cc: @kenwith

**Comment by TeresaFG-writer** [comment]
@joinimran Looks good.

**Pull request merged**